### DISCUSSION TITLE
Apply class only to a specific media selector

### DISCUSSION THREAD

**Discussion by mo-rocks** [tailwindlabs/tailwindcss#12246] [General]
If I have a flex box, is there a way (or could there be a way) to avoid this:
<label class="font-semibold w-10 self-center md:self-start lg:self-center">Name</label>

For the UI, on MD screens only, I need it to be self-start. It would be great if there was a way to indicate ONLY the md to use self-start, and set items-center on the parent, which would apply for all other screen sizes.

Is this possible?

**Comment by alex-krasikau** [comment]
Hello, @mdeblieck!

The simplest way to handle ONLY would be to use `md:{set-something} lg:{unset-something}`. In your case, it would be `md:self-start lg:self-auto`. You can see an example here: https://play.tailwindcss.com/T9B0oMSlBP

Alternatively, you can expand your configuration and add a `md-only` screen configuration like this:
```js
extend: {
   screens: {
     'md-only': {'min': '768px', 'max': '1023px'},
   },
}
```

A complete example can be found here: https://play.tailwindcss.com/3ua9KleyfL?file=config

**Comment by mo-rocks** [comment]
OK, thank you~
